src/cargo/core/compiler/fingerprint/dirty_reason.rs

```diff
@@ -183,6 +187,16 @@ impl DirtyReason {
             DirtyReason::PrecalculatedComponentsChanged { .. } => {
                 s.dirty_because(unit, "the precalculated components changed")
             }
+            DirtyReason::ChecksumUseChanged { old, new: _ } => {
```

Comment:
If we are not using `new` anywhere, should we just remove this field?

Reply:
Yeah, new is defined as precisely the opposite of `old`, so let's get rid of it. Done in ba5899ae0f2211184731b2a8a98313881cb85675